pyat/at/physics/linear.py

```diff
@@ -8,7 +8,6 @@
 from math import sqrt, pi, sin, cos, atan2
 
 import numpy
```

Comment:
While you are there, it would be better to use the [recommended way](https://numpy.org/doc/2.1/user/absolute_beginners.html#how-to-import-numpy) of importing numpy and use `np`.

Note: there was no problem with the 2 import lines. The import is done once, you just get 2 different names to refer to the same thing: quick and dirty trick to minimise the modifications.

Reply:
@lfarv rather that a recommendation this is a convention, there is absolutely no difference in effect.
Since the beginning of pyAT we used `import numpy` and not `np` as convention.

I have personally no preference but the convention has to be the same for the whole code, if you now want to switch to `import numpy as np` this has to be done for the whole code in a separate PR.

Reply:
At the moment, 39 AT modules use` import numpy`, and 14 modules use `import numpy as np` (mostly recent ones). So the situation is mixed. My preference for `np` comes from the fact that all the usage examples (in numpy documentation and in other sources) use `np`. This is not critical, so my suggestion is:
- use `np` for all new developments,
- preferably switch to `np` when significantly modifying an existing module.

This is similar to what we are doing with "black" formatting.